R/gcalendr_auth.R

```diff
@@ -63,7 +60,7 @@ gargle_lookup_table <- list(
 #' calendar_auth(path = "foofy-83ee9e7c9c48.json")
 #' }
 calendar_auth <- function(
-  email = NULL,
+  email = gargle::gargle_oauth_email(),
```

Comment:
It turns out to be important to set this default here, rather than further downstream. I learned that from working the example with Alex.